src/boot/bootctl.c

```diff
@@ -1647,6 +1653,18 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_arch_all = true;
                         break;
 
+                case ARG_EFI_LABEL:
+                        if (isempty(optarg) || !string_is_safe(optarg))
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                                       "Invalid --efi-label=: %s", strna(optarg));
```

Comment:
Please escape...
```
                        if (isempty(optarg) || !string_is_safe(optarg)) {
                                _cleanup_free_ char *escaped = NULL;

                                escaped = cescape(optarg);
                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                                       "Invalid --efi-label=: %s", strna(escaped));
                        }
```

Reply:
Sorry, I get it now... `strna` just prints out `n/a` if the string is NULL and doesn't handle escpaing.